parsuricata/_parser.py

```diff
@@ -71,12 +71,10 @@
 
     ?port_spec: variable
               | integer
-              | "[" port_grouping_spec ("," port_grouping_spec)* "]"    -> port_grouping
+              | port_range
+              | "[" port_spec ("," port_spec)* "]"                      -> port_grouping
               | "!" port_spec                                           -> negated
```

Comment:
These aliases really don't need to be indented so much, anymore. I think I've just been going by the rule "four spaces after the longest rule definition"
```suggestion
              | "[" port_spec ("," port_spec)* "]"    -> port_grouping
              | "!" port_spec                         -> negated
```